### DISCUSSION TITLE
`/filament/app.css` not loading after upgrading v2 to v3

### DISCUSSION THREAD

**Discussion by pekka** [filamentphp/filament#7482] [Help]
I've upgraded a v2 project to v3 but can't get the CSS to work, getting an unstyled page when logging into Filament. Looking at the page source, it appears that `/public/css/filament/forms/forms.css` and its neighbour `/public/css/filament/support/support.css`are getting loaded but `/public/css/filament/filament/app.css`is not.

When I manually force loading `app.css` in the page header everything works fine, but obviously that's bad as a permanent solution.

Anyone have an idea why this is, and where I should go look to rectify this? I think I followed all the upgrading steps correctly but of course can't rule out a mistake.

**Comment by pekka** [comment]
I had a call to a custom Vite theme in a service provider.

 `Filament::registerViteTheme('resources/css/app.less');`

**Reply by danharrin** [reply]
That's not a v3 thing really? Are you sure youre on v3?

**Reply by pekka** [reply]
That was a remnant from v2 in a service provider that appears to have clogged things up! All is well now 👍
